app/views/participants/scoreboard.haml

```diff
@@ -1,7 +1,19 @@
 - if @participants_exist
   .scoreboard
-    %button.btn.btn-primary.fullscreen
-      %i.fas.fa-expand
+    .d-flex.align-items-center
```

Comment:
This might be splitting hairs but I think you might want to use row/cols here instead? I think this is to keep the buttons grouped on the same line?

Reply:
It is, but my understanding with row/cols is to keep items evenly spaced/things that will span a whole row, whereas here it's just two buttons